adventurous-anteaters/adv_ant/retroNet/views.py

```diff
@@ -0,0 +1,106 @@
+from django.contrib.auth import login
+from django.http import HttpResponse
+from django.shortcuts import render, redirect
+from django.contrib.auth.forms import UserCreationForm
+from .models import Tweet, UpdateProfile
+from .forms import TweetForm, Updateprofile
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import logout
+
+
+# index view when user logs in
```

Comment:
Don't use floaty block comments to document functions - use docstrings.

This block comment isn't even over the right function.